### ISSUE TITLE
Text looks funny in Chrome when system UI font is set to Inter

### ISSUE THREAD

**Issue by hexchain** [swc-project/website#247]
User-agent: Chrome 124.0.6367.201 on Linux

The swc.rs website has the following CSS:

```css
html {
    /* ... */
    font-family: ui-sans-serif,system-ui,-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Noto Sans,Ubuntu,Cantarell,Helvetica Neue,Arial,sans-serif,Apple Color Emoji,Segoe UI Emoji,Segoe UI Symbol,Noto Color Emoji;
    /* ... */
    font-feature-settings: "rlig" 1,"calt" 1,"ss01" 1,"ss06" 1;
}
```

My system UI font is set to [Inter](https://rsms.me/inter/), whose `ss06` feature is "Characters in squares". This makes the texts in the page look funny:

![image](https://github.com/swc-project/website/assets/529520/9106a146-915c-425d-87c0-b61241da74b3)

**Comment by ankworld** [comment]
I encountered the same issue. Initially, I assumed it was a deliberate design choice of the website. However, after switching to a different font that was not "Inter-Fonts," the strange appearance of the website returned to normal. Upon further investigation, I discovered that the cause was the "font-feature-settings" ss06. I'm curious as to why they chose to enable ss06 since it makes reading the documentation difficult.
